notebooks/supernovae_plus_extended_source_tutorial.ipynb

```diff
@@ -0,0 +1,245 @@
+{
```

Comment:
<p>can we make the PSF not to have a negative element and hence not put out the lenstronomy warning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>